### DISCUSSION TITLE
Regular 2D Hexagonal

### DISCUSSION THREAD

**Discussion by Kareem-aboayanah** [neperfepx/neper#251] [Q&A] [answered]
Hi all Neper users,

I want to generate 2D regular hexagonal tessellations of the same size throughout the model. Is this possible in Neper and what command (if possible) generates them.

Thank you for your help!

Kareem Aboayanah
University of Toronto
![hex-tess](https://user-images.githubusercontent.com/85142578/122769704-cbaab880-d272-11eb-8a27-811093798ebb.png)

**Comment by dikshanshs** [comment, marked as answer]
@Kareem-aboayanah : You can try out setting equally spaced seeds and use -periodicity. You will get the same pattern.

**Reply by Kareem-aboayanah** [reply]
Thank  you so much!

**Reply by rquey** [reply]
Support for hexagons has been discontinued. This can be fixed in the future if there is interest.

**Reply by kmmm70** [reply]
Rquey,

I'm interested for doing that in 2D too. Please bring that feature back.

Thanks.

**Reply by Kareem-aboayanah** [reply]
It is definitely of interest. I use it for conceptual modeling of stress distribution in crystalline rocks. Thanks Rquey!
